editor.planx.uk/package.json

```diff
@@ -90,7 +90,7 @@
     "@testing-library/user-event": "^14.4.3",
     "@turf/helpers": "^6.5.0",
     "@types/draft-js": "^0.11.9",
-    "@types/jest": "^29.0.0",
+    "@types/jest": "^27.4.3",
```

Comment:
I wanted to use the more semantically correct `test.failing()` which my linter was allowing / suggesting (https://jestjs.io/docs/api#testfailingname-fn-timeout). 

This didn't work as our version of jest is controlled by `create-react-app` and not our own `package.json` 👍